real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/baseNodes/SerialNode.java

```diff
@@ -8,16 +10,18 @@
 import java.io.OutputStream;
 
 import com.orsoncharts.util.json.JSONObject;
-import com.roboclub.robobuggy.ros.Node;
 
 // Properly initializes the serial things
-public abstract class SerialNode implements Node {
+public abstract class SerialNode extends BuggyDecoratorNode {
+	
+	private final static int TIMEOUT = 2000;
+	//Need to manage real, simulated, and calculated sensors
 	
 	// Initialize first
 	String thread_name;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1804384399/issues/source?bid=2938665&fileBranchId=3059924#l21)